pkg/network/node/networkpolicy.go

```diff
@@ -148,7 +148,7 @@ func (np *networkPolicyPlugin) AddNetNamespace(netns *networkapi.NetNamespace) {
 	defer np.lock.Unlock()
 
 	if _, exists := np.namespaces[netns.NetID]; exists {
-		glog.Warning("Got AddNetNamespace for already-existing namespace %s (%d)", netns.NetName, netns.NetID)
+		glog.Warningf("Got AddNetNamespace for already-existing namespace %s (%d)", netns.NetName, netns.NetID)
```

Comment:
Isn't govet supposed to catch those?
...
OK, apparently, no, it looks for Logf, Errorf, and Fatalf, but not Warningf. We need to pass an argument. I'll file a bug about that.